src/sso/sso.spec.ts

```diff
@@ -6,7 +6,21 @@ import { WorkOS } from '../workos';
 describe('SSO', () => {
   describe('SSO', () => {
     describe('getAuthorizationURL', () => {
-      describe('with no custom api hostname', () => {
+      describe('with no custom api hostname and clientID', () => {
```

Comment:
I don't think we need to explicitly add "and clientID" to the `describe` blocks in the tests in this file.

`clientID` will be a required parameter and these tests are meant to enumerate on optional parameters.

Reply:
done! I also asked this question about the tests on Slack and got similar responses: https://work-os.slack.com/archives/CPEAGQU8G/p1608669941038900